Client/mods/deathmatch/logic/CClientModelManager.cpp

```diff
@@ -77,14 +70,14 @@ CClientModel* CClientModelManager::FindModelByID(int iModelID)
     return nullptr;
 }
 
-std::vector<CClientModel*> CClientModelManager::GetModelsByType(const eClientModelType type, const unsigned int minModelID)
+std::vector<std::shared_ptr<CClientModel>> CClientModelManager::GetModelsByType(eClientModelType type, const unsigned int minModelID)
```

Comment:
Nitpicky, but I'd say using a `ModeList_t` would be a better option here.
Or just move the return type to the rhs.